datadog/resource_datadog_dashboard.go

```diff
@@ -5649,6 +6079,154 @@ func buildDatadogGroupBySort(sort map[string]interface{}) *datadogV1.LogQueryDef
 	return ddSort
 }
 
+func buildTerraformQuery(datadogQueries []datadogV1.FormulaAndFunctionQueryDefinition) []map[string]interface{} {
+	queries := make([]map[string]interface{}, len(datadogQueries))
+	for i, query := range datadogQueries {
+		terraformQuery := map[string]interface{}{}
+		if query.TimeSeriesFormulaAndFunctionEventQueryDefinition != nil {
+			if dataSource, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetDataSourceOk(); ok {
+				terraformQuery["data_source"] = dataSource
+			}
+			if name, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetNameOk(); ok {
+				terraformQuery["name"] = name
+			}
+			if indexes, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetIndexesOk(); ok {
+				terraformQuery["indexes"] = indexes
+			}
+			if search, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetSearchOk(); ok {
+				if len(search.GetQuery()) > 0 {
+					terraformSearch := map[string]interface{}{}
+					terraformSearch["query"] = search.GetQuery()
+					terraformSearchList := []map[string]interface{}{terraformSearch}
+					terraformQuery["search"] = terraformSearchList
+				}
+			}
+			if compute, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetComputeOk(); ok {
+				terraformCompute := map[string]interface{}{}
+				if aggregation, ok := compute.GetAggregationOk(); ok {
+					terraformCompute["aggregation"] = aggregation
+				}
+				if interval, ok := compute.GetIntervalOk(); ok {
+					terraformCompute["interval"] = interval
+				}
+				if metric, ok := compute.GetMetricOk(); ok {
+					terraformCompute["metric"] = metric
+				}
+				terraformComputeList := []map[string]interface{}{terraformCompute}
+				terraformQuery["compute"] = terraformComputeList
+			}
+			if v, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetGroupByOk(); ok {
+				terraformGroupBys := make([]map[string]interface{}, len(query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetGroupBy()))
```

Comment:
can this just be range of `v`?

Reply:
Addressed in [commit](https://github.com/DataDog/terraform-provider-datadog/pull/892/commits/7dc2cf236121ffaabd23f38a0686743f6e0f8640).